codalab/lib/worksheet_util.py

```diff
@@ -678,10 +678,11 @@ def genpath_to_target(bundle_info, genpath):
         # strip off the leading / from genpath to create a subpath in the target.
         return (bundle_info['uuid'], genpath[1:])
 
-    def flush_bundles():
+    def flush_bundles(bundle_block_start_index):
```

Comment:
Maybe more descriptive to call it `display_mode_start_index`

Reply:
It points to the first []{0x123} for bundle blocks even if there is a % display above since default schema doesn't require such line